rotkehlchen/serialization/deserialize.py

```diff
@@ -380,24 +381,36 @@ def deserialize_trade_pair(pair: str) -> TradePair:
     return TradePair(pair)
 
 
-def deserialize_asset_movement_category(symbol: str) -> AssetMovementCategory:
+def deserialize_asset_movement_category(
+    value: Union[str, HistoryEventType],
+) -> AssetMovementCategory:
     """Takes a string and determines whether to accept it as an asset movement category
 
-    Can throw DeserializationError if symbol is not as expected
+    Can throw DeserializationError if value is not as expected
     """
-    if not isinstance(symbol, str):
+    if not (isinstance(value, str) or isinstance(value, HistoryEventType)):
```

Comment:
isinstance can do the same in one call
```suggestion
    if not isinstance(value, (str, HistoryEventType)):
```